src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/ThemeChange.cs

```diff
@@ -0,0 +1,32 @@
+#if ANDROID || IOS
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class ThemeChange : _IssuesUITest
+	{
+		public override string Issue => "UI theme change during the runtime";
+
+		public ThemeChange(TestDevice device) : base(device)
+		{
+		}
+
+		[Test]
+		[Category(UITestCategories.LifeCycle)]
```

Comment:
Didnt want to add a new category just for this so with a little imagination this fits in LifeCycle...?

Reply:
Yeah, imo it's resonable :)